strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/AuthorizationConfigController.java

```diff
@@ -36,109 +48,90 @@
         }
-        else
+        if (bindingResult.hasErrors())
         {
-            return toError("Unable to locate AuthorizationConfig to update...");
+            throw new RequestBodyValidationException(FAILED_ADD_ROLE, bindingResult);
         }
-    }
 
-    // ----------------------------------------------------------------------------------------------------------------
-    // Add role
+        // Convert RoleForm to Role
+        Role role = conversionService.convert(roleForm, Role.class);
 
-    @ApiOperation(value = "Used to add new roles")
-    @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = "The role was created successfully."),
-                            @ApiResponse(code = 400,
-                                         message = "An error occurred.") })
-    @RequestMapping(value = "role",
-                    method = RequestMethod.POST,
-                    consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity addRole(@RequestBody Role role)
-            throws JAXBException
+        return processConfig(config -> addRole(config, role), () -> SUCCESSFUL_ADD_ROLE, acceptHeader);
+    }
+
+    private void addRole(AuthorizationConfig config,
+                         Role role)
     {
 
-        logger.info("Trying to add new role \n" + role);
-        logger.debug(role.toString());
-        return processConfig(config ->
-                             {
-                                 //  Role role = read(json, Role.class);
-                                 boolean result = config.getRoles()
-                                                        .getRoles()
-                                                        .add(role);
-
-                                 if (result)
-                                 {
-                                     configProvider.updateConfig(config);
-                                     logger.info("Successfully added new role " + role.getName());
-                                 }
-                                 else
-                                 {
-                                     logger.warn("Unable to add new role " + role.getName());
-                                 }
-                             });
+        boolean result = config.getRoles()
+                               .getRoles()
+                               .add(role);
+
+        if (result)
+        {
+            configProvider.updateConfig(config);
+        }
     }
 
     // ----------------------------------------------------------------------------------------------------------------
     // View authorization config as XML file
 
     @ApiOperation(value = "Retrieves the strongbox-authorization.xml configuration file.")
     @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = ""),
+                                         message = "Everything went ok."),
```

Comment:
`message = "Everything went ok."`

Could you please use constant value as you did in previous method ? Please stay consistent :)